kustomize/README.md

```diff
@@ -125,10 +53,39 @@ Alternatively, if you would like to quickly provision a GKE cluster, you can use
     shippingservice-6ccc89f8fd-v686r         1/1     Running   0          2m58s
     ```
 
+    _Note: It may take 2-3 minutes before the changes are reflected on the deployment._
+
 1. Access the web frontend in a browser using the frontend's `EXTERNAL_IP`.
 
     ```
     kubectl get service frontend-external | awk '{print $4}'
     ```
 
     Note: you may see `<pending>` while GCP provisions the load balancer. If this happens, wait a few minutes and re-run the command.
+
+
+## Deploy Online Boutique variations with Kustomize
+
+Here is the list of the variations available as Kustomize components that you could leverage:
+
+- [**Cymbal Shops Branding**](/components/cymbal-branding)
+  - Changes all Online Boutique-related branding to Google Cloud's fictitious company — Cymbal Shops. The code adds/enables an environment variable `CYMBAL_BRANDING` in the `frontend` service.
+- [**Google Cloud Operations**](/components/google-cloud-operations)
+  - Enables Monitoring (Stats), Tracing, Profiler, and Debugger for various services within Online Boutique. The code removes the existing environment variables (`DISABLE_STATS`, `DISABLE_TRACING`, `DISABLE_PROFILER`, `DISABLE_DEBUGGER`) from appropriate YAML config files.
+- [**Memorystore (redis)**](/components/memorystore)
+  - The default Online Boutique deployment uses the default in-cluster `redis` database for storing the contents of its shopping cart. The Memorystore deployment variation overrides the default database with its own Memorystore (redis) database. These changes directly affect `cartservice`.
+- [**Network Policies**](/components/network-policies)
```

Comment:
**Reminder:**
I assume you'll eventually add a short description for "Network Policies".
But adding this comment as a reminder for both of us. :)

Reply:
Yep, already locally, will be there soon ;)

Reply:
Done.